bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/data/EventSubType.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.netatmo.internal.api.data;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * This enum describes sub events in relation to a given event
+ *
+ * @author Gaël L'hopital - Initial contribution
+ */
+@NonNullByDefault
+public enum EventSubType {
+    SD_CARD_MISSING(EventType.SD, 1),
+    SD_CARD_INSERTED(EventType.SD, 2),
+    SD_CARD_FORMATED(EventType.SD, 3),
```

Comment:
Still typo: SD_CARD_FORMAT**T**ED

Reply:
Yes, this is corrected on my side, I'll push it once I have more review from @lolodomo - which is sure will happen :)

Reply:
In meantime, maybe you could have a look at this? :) https://github.com/openhab/openhab-addons/pull/10831#issuecomment-987142204

Reply:
Sure it will ;) 
Please start by adding a description to all your classes, this will avoid many review comments from myself ;)

Reply:
Done